topjohnwu/ndk-busybox-kitchen
# Update for BusyBox 1.34.1, Fix build for latest pcre2, Fix UB from clang busybox optimization

Update for BusyBox 1.34.1
- remove upstreamed patches
- combine leftover hush platform patches
- rebase patches over released stable 1.34.1

Fix build for latest pcre2
- update external.mk for pcre2 source reorganization
- work around a new symlink from .c -> .c.dist which can break NDK compiles on Windows

Fix UB from clang busybox optimization
- see https://github.com/topjohnwu/Magisk/pull/4731
- thanks @yujincheng08 and @canyie for the upstream busybox patches
